js/apps/system/_admin/aardvark/APP/react/src/views/g6graphs/AddEdgeModal.js

```diff
@@ -126,7 +131,7 @@ export const AddEdgeModal = ({ edgeModelToAdd, shouldShow, onUpdateEdge, onReque
           }
         </div>
         <div style={{ 'marginTop': '38px', 'textAlign': 'right' }}>
-          <StyledButton className="button-close" onClick={onRequestClose}>Cancel</StyledButton>
+          <StyledButton className="button-close" onClick={() => cancelEdge()}>Cancel</StyledButton>
```

Comment:
```suggestion
          <StyledButton className="button-close" onClick={cancelEdge}>Cancel</StyledButton>
```

This can simple be a ref